go-controller/pkg/node/vrfmanager/vrf_manager.go

```diff
@@ -16,18 +17,21 @@ type vrf struct {
 	name       string
 	table      uint32
 	interfaces sets.Set[string]
+	routes     []netlink.Route
 	delete     bool
 }
 
 type Controller struct {
-	mu   *sync.Mutex
-	vrfs map[string]*vrf
+	mu           *sync.Mutex
+	vrfs         map[string]*vrf
+	RouteManager *routemanager.Controller
```

Comment:
make it as a private variable

Reply:
It's needed for Unit tests.. hence its public